modules/behaviors/DataBehavior.js

```diff
@@ -220,6 +220,8 @@
 
       Behavior.apply(this, arguments);
 
+      this.set('loadingIds', 0);
```

Comment:
Any reason for using 0 instead of true/false?

Reply:
yes for simultaneous requests.  If one finishes you don't want to say you are done if there is still another one in progress.

Reply:
oh I see